server/index.js

```diff
@@ -19,11 +24,32 @@ app.get('/houses/*', (req, res) => {
           res.send(rows); 
         })
         .then((res) => {
-          console.log(res);
+          //console.log(res);
+          conn.end();
+        })
+        .catch(err => {
+          //console.log(`Error: ${err}`);
+          conn.end();
+        })
+    }).catch(err => {
+      console.log(`Not connected to MariaDB: ${err}`);
+    });
+  });
+
+  app.get('/test', (req, res) => {
```

Comment:
There should not be a route specifically for testing. Instead, figure out some way to test your routes.